src/components/Rubric/hooks.test.ts

```diff
@@ -0,0 +1,117 @@
+import { mocked } from 'ts-jest/utils';
+
+import {
+  usePageData,
+  useRubricConfig,
+} from 'data/services/lms/hooks/selectors';
+import { mockUseKeyedState } from '@edx/react-unit-test-utils';
+import { submitRubric } from 'data/services/lms/hooks/actions';
+import { useRubricData, stateKeys } from './hooks';
+
+jest.mock('data/services/lms/hooks/selectors');
+jest.mock('data/services/lms/hooks/actions');
+
+const mockedUsePageData = mocked(usePageData);
```

Comment:
We are using `jest-when` for this in the api code.  Is there a reason to have a different standard here? if not, I would rather keep the convention for consistency.

Reply:
this isn't a new pattern. It is a remanent of using typescript.